nntrainer/tensor/manager.h

```diff
@@ -111,10 +111,7 @@ class Manager {
   /**
    * @brief     Constructor of Manager
    */
-  Manager(bool enable_gradient_memory_opt_ = true,
-          bool enable_derivative_memory_opt_ = false,
-          bool enable_activation_memory_opt_ = false,
-          bool enable_inference_inout_memory_opt_ = false);
+  Manager() = default;
```

Comment:
static analyzer is likely to unlike this. (I think it was because of some primitive default initializers)

Reply:
All manager's members are now either vector or TensorPool and both have good default constructors, so I think it will be fine. If this causes issues, I will change.